app/controllers/assessments_controller.rb

```diff
@@ -145,7 +145,6 @@ def update
     redirect_to(root_url) && return if patient.nil?
 
     redirect_to root_url unless current_user&.can_edit_patient_assessments?
-    patient = Patient.find_by(submission_token: params.permit(:patient_submission_token)[:patient_submission_token])
```

Comment:
Good catch. Commenting for reviewer awareness. Until this change goes in, patients that somehow had the old submission token wouldn't have their assessments updated properly by users. I don't actually think this is a possible case given the migration that was done when switching to newer submission tokens, but good to remove nonetheless.